doc/Makefile.am

```diff
@@ -18,7 +18,8 @@
 # GregorioRef.tex must be the first source file
 SRCFILES = GregorioRef.tex Command_Index_gregorio.tex \
 		   Command_Index_internal.tex Command_Index_User.tex \
-		   Gabc.tex Appendix_Font_Tables.tex GregorioRef.lua factus.gabc
+		   Gabc.tex Appendix_Font_Tables.tex GregorioRef.lua factus.gabc \
+		   scorefonttable.lua extrafonttable.lua
 NABCSRCFILES = GregorioNabcRef.tex veni.gabc
```

Comment:
There are a couple of other places where the new files may needed to be added to manifests.  Off-hand, I think the installers for Mac and Windows might need them (though, since the new files are lua file, they may be picked up by wild cards).  You should also check `install-gtex.sh`.